ios/REANodesManager.m

```diff
@@ -105,13 +143,35 @@ - (void)onAnimationFrame:(CADisplayLink *)displayLink
   }
 
   [REANode runPropUpdates:_updateContext];
+  NSMutableArray<REANativeUpdateOperation *> *copiedProps = [_operationsInBatch copy];
+  int copiedNumberOfOperationsInBatch = _numberOfOperationsInBatch;
+  [_operationsInBatch removeAllObjects];
+  _numberOfOperationsInBatch = 0;
+  if (copiedNumberOfOperationsInBatch != 0) {
+    RCTExecuteOnUIManagerQueue(^{
+      BOOL shouldFinishBatch = ((NSHashTable<RCTShadowView *> *)[self.uiManager valueForKey:@"_shadowViewsWithUpdatedChildren"]).count == 0;
+      for (int i = 0; i < copiedNumberOfOperationsInBatch; i++) {
+        [self.uiManager updateView:copiedProps[i].reactTag viewName:copiedProps[i].viewName props:copiedProps[i].nativeProps];
+      }
+      if (shouldFinishBatch) {
+        [self.uiManager batchDidComplete];
```

Comment:
Maybe we can use https://github.com/facebook/react-native/blob/310a2850c55fef194d54fcb7ea4649ed0e800deb/React/Modules/RCTUIManager.m#L1132